pkg/omf/functions/cli/omf.cli.search.fish

```diff
@@ -0,0 +1,26 @@
+function omf.cli.search -d "CLI parser for the search command"
+
+  switch (count $argv);
+    case 1;
+      omf.search.pkg $argv[1]
+      echo
+      omf.search.theme $argv[1]
+    case 2;
+      switch "$argv[1]"
+        case "-pkg" "--package";
+          omf.search.pkg $argv[2]
+        case "-t" "--theme";
+          omf.search.theme $argv[2]
+        case '*';
+          __omf.search.usage
```

Comment:
This is a typo I think; it should be `__omf.cli.search.usage` instead of `__omf.search.usage`.